example/build.gradle

```diff
@@ -0,0 +1,46 @@
+apply plugin: 'com.android.library'
```

Comment:
With the standard `apt` plugin, the Android Studio still could not mark the generated .java file as source file, but the command line build has no problem, and the generated jar file don't contain .java files anymore. Maybe it's a bug of Android Studio?